### ISSUE TITLE
New web Page Request

### ISSUE THREAD

**Issue by Sayed-afridy** [meeting-room-booking-system/mrbs-code#3516]
is it possible to get something like this?? i want to add panels infront of rooms/desk 
![conference-room-reservation-screen](https://github.com/meeting-room-booking-system/mrbs-code/assets/85822573/5fb6f445-be6e-450f-bb28-d3bc72268b15)

**Comment by Sayed-afridy** [comment]
> Have you looked at kiosk mode in MRBS?

yes i have tried the kiosk mode .. but not much information there. also i cant set kiosk for multiple rooms on multiple devices . 
also there is nothing showing that meeting is in progress

**Comment by campbell-m** [comment]
> also i cant set kiosk for multiple rooms on multiple devices

Do you mean having a display outside Room A showing the bookings for Room A and another outside Room B showing the bookings for Room B?  If so, yes you can.

**Comment by Sayed-afridy** [comment]
> > also i cant set kiosk for multiple rooms on multiple devices
> 
> Do you mean having a display outside Room A showing the bookings for Room A and another outside Room B showing the bookings for Room B? If so, yes you can.

can you tell me the procedure as i am only able to set the kiosk mode once. and is there anyway to get similar kind of dashboard attached earlier

**Comment by campbell-m** [comment]
> an you tell me the procedure as i am only able to set the kiosk mode once.

Log in to the Room A device and enter Kiosk mode.  Then login to the Room B device and enter Kiosk mode.

**Comment by campbell-m** [comment]
> and is there anyway to get similar kind of dashboard attached earlier

Not at the moment.  See also Issue #3506.
